libnymea-networkmanager/networkmanager.cpp

```diff
@@ -154,6 +154,8 @@ NetworkManager::NetworkManagerError NetworkManager::connectWifi(const QString &i
     connectionSettings.insert("id", ssid);
     connectionSettings.insert("uuid", QUuid::createUuid().toString().remove("{").remove("}"));
     connectionSettings.insert("type", "802-11-wireless");
+    connectionSettings.insert("autoconnect-retries", 0); // 0 = forever, -1 = default (4 retries)
```

Comment:
are you reading this somewhere (not in this diff)

seems to me like 0 is default not 4 ;), IMHO the -1 is confusing and should just be dropped. If someone wants 4, he should use 4, not -1. 
Also, what about 0 retries (as in, try once and give up), not possible any more if 0 means forever, no?

Reply:
This is network-manager value logic: `-1` is default and means: retry 4 times and then give up.
`0` means retry forever to connect to this network if available. Any number > 0 tells the network-manager to retry that amount and then stop